games-strategy/lgeneral/lgeneral-1.3.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A turn-based strategy engine"
+DESCRIPTION="LGeneral is a turn-based strategy engine heavily inspired by Panzer General. \
+You play single scenarios or whole campaigns turn by turn against a human \
+player or the AI. Entrenchment, rugged defense, defensive fire, surprise \
+contacts, surrender, unit supply, weather influence, reinforcements and other \
+implementations contribute to the tactical and strategic depth of the game."
+HOMEPAGE="http://lgames.sourceforge.net/"
+COPYRIGHT="2000, 2015 Michael Speck, Leo Savernik, Peter Ivanyi"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/lgeneral/lgeneral/lgeneral-$portVersion.tar.gz"
```

Comment:
BTW, if you want can shorten it a little:
```
SOURCE_URI="http://downloads.sourceforge.net/lgeneral/lgeneral-$portVersion.tar.gz"
```
(and even a little bit more if you also replace **`downloads.sourceforge.net`** by **`downloads.sf.net`**)
This has the advantage of "allowing" upstream to move the archive from one directory to another, e.g. to "OLD" subdirectories, without breaking the short URI since the mirrors at SourceForge always reply with the up-to-date "target" URI.